README.md

```diff
@@ -86,6 +87,12 @@ var options = {
 cy.layout( options ).run();
 ```
 
+You can set layout options per node by defining a `nodeLayoutOptions` function which is called on a per-node basis. This is useful for tweaking the layout of a particular node, like for [setting its partition](https://www.eclipse.org/elk/reference/options/org-eclipse-elk-partitioning-partition.html) for the layered layout.
+For instance, if you want to store these options within the node's `scratch` object, you can do something like this:
+
+```js
+nodeLayoutOptions: (n: any) => n.scratch('layoutOptions')
```

Comment:
```suggestion
nodeLayoutOptions: node => n.scratch('layoutOptions')
```

Reply:
The consumer might not be using TS.  What do you think about a change like this?